src/Schema/Blueprint.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Saritasa\Database\Eloquent\Schema;
+
+use Closure;
+use Illuminate\Database\Schema\Blueprint;
+
+class CamelCaseBlueprint extends Blueprint
```

Comment:
1. Class name doesn't match file name (PSR-4 violation) - this should have not pass `make cs`.
2. Let's name it `BlueprintCamelCase`.
3. Add class comment - description, how it is different from base Blueprint (you can copy from README).

Reply:
Try to install and use locally: https://stackoverflow.com/questions/13498519/how-to-require-a-fork-with-composer